docs/_src/api/openapi/openapi.json

```diff
@@ -1 +1,739 @@
\ No newline at end of file
+{
```

Comment:
Just as a warning. Watch out when merging this PR with your other changes of `openapi.json` in other open PRs.

Reply:
I'll manually check if anybody has merge conflicts, hopefully there won't be many